tests/suites/test_suite_ssl.function

```diff
@@ -3014,13 +3062,20 @@ exit:
 /* END_CASE */
 
 /* BEGIN_CASE depends_on:MBEDTLS_X509_CRT_PARSE_C:!MBEDTLS_USE_PSA_CRYPTO:MBEDTLS_PKCS1_V15 */
-void handshake( const char *cipher, int version, int pk_alg )
+void handshake( const char *cipher, int version, int pk_alg,
+                const char *psk_str )
```

Comment:
COnsider using `data_t` instead of `const char *` for automatic dehexing.

Reply:
Done. Much clearer now.